18F/identity-idp
# LG-10456: Update loading content for Face/Touch setup

## 🎫 Ticket

[LG-10456](https://cm-jira.usa.gov/browse/LG-10456)

## 🛠 Summary of changes

Updates content of the Face or Touch Unlock setup screen to match design specifications.

## 📜 Testing Plan

1. Go to http://localhost:3000
2. Create an account
3. Select Face or Touch Unlock at MFA setup screen
4. Click Continue
5. Enter a device nickname
6. Click Continue
7. Observe that content matches the Figma design (see ticket for reference)

## 👀 Screenshots

Before|After
---|---
![Screen Shot 2023-08-01 at 9 08 06 AM](https://github.com/18F/identity-idp/assets/1779930/74f485ae-a5f7-4726-9ff5-226a9e41061f)|![Screen Shot 2023-08-01 at 9 05 43 AM](https://github.com/18F/identity-idp/assets/1779930/31da203f-d1a2-4e1f-bb68-0afd45796cbf)
